contrib/addr.py

```diff
@@ -8,12 +8,20 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--testnet', action='store_true')
+    parser.add_argument('--regtest', action='store_true')
+    parser.add_argument('--signet', action='store_true')
     parser.add_argument('address', nargs='+')
     args = parser.parse_args()
 
     if args.testnet:
         port = 60001
         from pycoin.symbols.xtn import network
+    if args.regtest:
+        port = 60401
+        from pycoin.symbols.xrt import network
+    if args.signet:
+        port = 60601
+        from pycoin.symbols.xtn import network
```

Comment:
Would be nice to explicitly check if only one is specified. Although not important since it's a simple example.